README.md

```diff
@@ -32,7 +32,7 @@ With the arrow keys (left and right) you can navigate through the build order. H
 
 ## Requirements
 
-[Node 10 or newer](https://nodejs.org/en/download/) is required to create this static website.
+[Node 12 or newer](https://nodejs.org/en/download/) is required to create this static website.
```

Comment:
Are you sure that node 10 does not work anymore?
The node 10 runner still seems to work 
https://github.com/BurnySc2/sc2-planner/runs/5963405472?check_suite_focus=true
https://github.com/BurnySc2/sc2-planner/blob/532023b7bec78bde03ac88be538622c648fefbca/.github/workflows/react.yml#L11

Reply:
I originally tried running it with node10 and it failed because Object.fromEntries() isn't defined.  At the bottom of this, it shows that it was added in node12.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries
Updating to node17 fixed it for me, though it introduced a different issue with openssl that I made a note for you here #42

Reply:
Thank you for the detailled answer!